foundation-extensions/identity-for-edge-network/api-reference.md

```diff
@@ -1,5 +1,61 @@
 # API Reference
 
+## extensionVersion
+
+{% tabs %}
+{% tab title="Android" %}
+### extensionVersion
+
+The extensionVersion() API returns the version of the identity for edge network extension that is registered with the Mobile Core extension.
```

Comment:
For all the tabs in this file, let's remove the API name from the tab as it just repeats the API name above the tab group. Move common API descriptions above the tab group as well to reduce clutter in the tabs. Any platform-specific information can be kept in the tab for that platform. 
You can look at how the Identity api reference was restructured for reference: https://aep-sdks.gitbook.io/docs/foundation-extensions/mobile-core/identity/identity-api-reference